canvas_cli/apps/plugin/plugin.py

```diff
@@ -133,6 +138,29 @@ def init(
     print.json(f"Project created in {project_dir}", project_dir=project_dir)
 
 
+@app.command(short_help="Validates the Canvas Manifest json file of a plugin package")
+def validate(
+    package: Path = typer.Argument(
+        ..., help="Path to a dir containing the python package to install"
+    ),
+) -> None:
+    """Validates the Canvas Manifest json file of a plugin directory."""
+    if not package.exists():
+        raise typer.BadParameter(f"Package {package} does not exist")
+
+    if not package.is_dir():
+        raise typer.BadParameter(f"Package {package} is not a directory, nothing to validate")
+
+    manifest = Path(f"{package.name}/CANVAS_MANIFEST.json")
+    if not manifest.exists():
+        raise typer.BadParameter(
+            f"Package {package} does not have a CANVAS_MANIFEST.json file to validate"
+        )
+
+    validate_manifest_file(json.loads(manifest.read_text()))
```

Comment:
It might be a good idea to raise an error is there is invalid json in the file. An explicit message saying that the json is invalid might be more helpful to the user than raising a JSONDecodeError.